browser/Makefile.am

```diff
@@ -1083,6 +1084,7 @@ pot:
 		src/app/DocEvents.ts \
 		src/app/GraphicSelectionMiddleware.ts \
 		src/errormessages.js \
+		src/layer/tile/CanvasTileWorker.js \
```

Comment:
I don't know that it's necessary to add this here, I don't really have an adequate understanding of the build system and how cool gets packaged. Advice welcome :)

Reply:
COOL_JS_LST is all the javascript files, so needs to be in there

The pot target is the gettext target for extracting translations. So I imagine it doesn't need to be in that list

(no idea why there is a separate list and not just extract over all COOl_JS_LST for simplicity)